cfgov/templates/admin/form_templates/info-unit-form.html

```diff
@@ -0,0 +1,18 @@
+<div class="{{ classname }}">
```

Comment:
What is this new template for and how does it relate to the sidefoot_heading changes?